src/services/dashboards/widgets/aws-personal-health-dashboard/widget-config.ts

```diff
@@ -0,0 +1,22 @@
+import type { WidgetConfig } from '@/services/dashboards/widgets/_configs/config';
+
+const awsPersonalHealthDashboardWidgetConfig: WidgetConfig = {
+    widget_config_id: 'awsPersonalHealthDashboard',
+    title: 'AWS Personal Health Dashboard',
+    base_configs: [{ config_id: 'dashboardCommon' }],
```

Comment:
todo: `filters.provider` is not in this widget, but `dashboardCommon` config has it.
In this case, you have to use `excludePropertiesFromDashboardCommonWidgetConfig()` (what a long name...😅)
Please look `aws-data-transfer-by-region/widget-config.ts`!